client/src/containers/section/page/networks/network-detail-parsers.ts

```diff
@@ -0,0 +1,132 @@
+  return typeof field !== 'undefined' && field?.data;
+};
+
+export const getProjectFields = (dataWithType: ProjectWithType) => {
+  const {
+    start_date: startDate,
+    end_date: endDate,
+    country_of_coordination: countryOfCoordination,
+    project_coordinator_name: projectCoordinatorName,
+    project_coordinator_email: projectCoordinatorEmail,
+    second_project_coordinator_name: secondProjectCoordinatorName,
+    second_project_coordinator_email: secondProjectCoordinatorEmail,
+    project_type: projectType,
+    country_of_interventions: countryOfInterventions,
+    main_area_of_intervention: mainAreaOfIntervention,
+    secondary_area_of_intervention: secondaryAreaOfIntervention,
+    third_area_of_intervention: thirdAreaOfIntervention,
+    main_area_of_intervention_other: mainAreaOfInterventionOther,
+    sustainable_development_goal: sustainableDevelopmentGoal,
+  } = dataWithType.attributes || {};
+
+  const fields = [];
+  if (typeof startDate !== 'undefined') {
+    const formatDate = (date: string) =>
+      format({
+        id: 'formatDate',
+        value: date,
+      });
+
+    fields.push({
+      label: 'Dates',
+      value: `${formatDate(startDate)}${endDate ? ` - ${formatDate(endDate)}` : ''}`,
+    });
+  }
+  if (hasData(countryOfCoordination)) {
+    fields.push({
+      label: 'Country of coordination',
+      value: countryOfCoordination?.data?.attributes?.name,
+    });
+  }
+  if (hasData(projectCoordinatorName)) {
+    if (hasData(secondProjectCoordinatorName)) {
+      fields.push({
+        label: 'Project coordinators',
+        value: [projectCoordinatorName, secondProjectCoordinatorName],
+        url: [`mailto:${projectCoordinatorEmail}`, `mailto:${secondProjectCoordinatorEmail}`],
+      });
+    } else {
+      fields.push({
+        label: 'Project coordinator',
+        value: projectCoordinatorName,
+        url: `mailto:${projectCoordinatorEmail}`,
+      });
+    }
+  }
+  if (hasData(projectType)) {
+    fields.push({
+      label: 'Project type',
+      value: projectType?.data?.attributes?.name,
+    });
+  }
+  if (hasData(countryOfInterventions) && countryOfInterventions?.data?.length) {
+    fields.push({
+      label: 'Country of interventions',
+      value: countryOfInterventions?.data?.map((c) => c.attributes?.name).join(', '),
+    });
+  }
+  if (hasData(mainAreaOfIntervention)) {
+    if (mainAreaOfIntervention?.data?.attributes?.name === 'Other (to be specified)') {
+      fields.push({
+        label: 'Main area of interventions',
+        value: mainAreaOfInterventionOther,
+      });
+    } else {
+      const mainAreaOfInterventions = [
+        mainAreaOfIntervention?.data?.attributes?.name,
+        secondaryAreaOfIntervention?.data?.attributes?.name,
+        thirdAreaOfIntervention?.data?.attributes?.name,
+      ]
+        .filter(Boolean)
+        .join(', ');
+      fields.push({
+        label: 'Main area of interventions',
```

Comment:
Can we rename the field?
```suggestion
        label: 'Main areas of interventions',
```